### ISSUE TITLE
Release Timetable

### ISSUE THREAD

**Issue by Arhowk** [Arhowk/Banjoball#13]
Closed Alpha (Target: Soon^TM)
- [ ] Hero Selection working in its current state (Arhowk)
- [ ] UI done for Q ability & mana indicator (Arhowk)
- [ ] UI for score done (Arhowk)
- [ ] Sunny Hills done & beautiful (Dayne)

Closed beta (Target: Christmas)
- [ ] Garf Physics done (Garfield)
- [ ] Minimap, team indicator UIs done (Arhowk)
- [ ] Hero Selection reorganized per category (Arhowk)
- [ ] multiple new heroes  (Dayne)

Friends-only Beta (Target: January)
- [ ] Physics bug fixes, optimizations (Garfield)
- [ ] All existing heroes rewritten to be compatible with physics engine (Garfield & Dayne)
- [ ] Flair selection done (Arhowk)
- [ ] Taunts implemented (Arhowk)

Open Beta
- [ ] Team Selection screen done (Arhowk)
- [ ] New heroes & bug fixes (Garfield & Dayne)
- [ ] New map for 8v8 (Dayne)

**Comment by CarpeSwag** [comment]
I stopped trying to work on the terrain and opted for coding a new hero. Terrain is gonna be a tooootal bitch if we want a hilly terrain it seems.

**Comment by Arhowk** [comment]
just render a displaced mesh above the tile grid.... its stupid that the tile grid cant do hills but meshing isnt hard

**Comment by Arhowk** [comment]
gave u another closed alpha job :dancer:

**Comment by CarpeSwag** [comment]
Yeah I mean that was my plan, but I wasn't sure if trees were gonna work with that or if there was any way to not get a lazy river.

I'll start working on that shit though I guess today. 

Btw, add game settings to alpha so hosts can set timed mode and score (and how long for both)

**Comment by Arhowk** [comment]
ugh that requires me making the damnn team selection scoreboard though

**Comment by CarpeSwag** [comment]
If that's the case then feel free to move it back towards later, but the sooner we have it the better IMO.

**Comment by Arhowk** [comment]
Meh. I don't see a real purpose to have it before open beta. Friends-only alpha will probs only be 1st-to-13

**Comment by schutzekatze** [comment]
I like how Arhowk added bugfixes and optimizations as an additional clause to my work. Thanks for having faith in me. :^)

**Comment by Arhowk** [comment]
Love u too boo 😘

**Comment by Arhowk** [comment]
pfft you cant mark off "new heroes" as done, we need atleast 10 more new heroes

**Comment by CarpeSwag** [comment]
Nah fam the new heroes for beta is done. I don't want to release too many untested heroes it'll be a shitshow for balancing.

We have 6 more heroes than Strikers atm, and 3 of them are completely new.

**Comment by Arhowk** [comment]
are completely sucky*

I'll make oracle and trickster for the closed beta as well

**Comment by CarpeSwag** [comment]
Nah fam we shouldn't overload the hero count.

**Comment by Arhowk** [comment]
Nah fam. Closed beta will be testing them all. Open releases will have them
locked based on rank.

**Comment by CarpeSwag** [comment]
Wait we're moving heroes based on rank? That's gonna turn away players. Also we can easily skip a closed beta and just release an open beta with what we have + UI atm.

**Comment by Arhowk** [comment]
No. We should be communicating with the veteran community before committing
any changes to the public.

If we have a good pool of heroes, we can put some of the best heroes in the
free hero pool. We can also do it CoD style where every 10 ranks, you get
to unlock one of a pool of the 15 or so heroes we have Available than keep
like pudge, rubick, or oracle as achievement or high level unlocks.

The key purpose behind this is to encourage people to continually play
public matches instead of getting bored of pub stomping and quitting

**Comment by CarpeSwag** [comment]
I've already communicated with parts of the veteran community lol. Some of them have already tested out some of the heroes tbh. Point is we can ship straight to beta with the content we have as is. If we ship with two other completely new heroes, then we're forcing ourselves to delay open beta longer for no real gain (we can test your heroes out via the testing mode mod and get feedback pre release for those)

I get the idea behind gating content, but I disagree with it being on heroes. At the very least, let's not start off with open beta being gated. Let's move gating to a month or two down the line from then.

**Comment by Arhowk** [comment]
If we don't gate immediately, it could have adverse affects by locking out
heroes that the public is used to playing. (Not saying gating right off of
the bat doesn't do this, but I'd say there's enough new stuff to help keep
people entertained through a few ranks)

**Comment by CarpeSwag** [comment]
Yah I meant to address that too, but pressed comment too early. I edited in additional comments.

> Also we should give time for players to get games in into our logs before we set it up so that once we set up the gating finally, it's only gating new players rather than everyone.

> No returning player will be happy if one day they just lose access to half the heroes and are told they have to grind more games. Basically we should just ease the playerbase into it rather than slapping them in the face with it off the bat.

**Comment by Arhowk** [comment]
Sorry I'm using email.

We could leave it as is for a while but keep track of rankings and whatnot
and notify the user base of the changes at an upcoming date, e.g. two weeks
or something to get ranked up enough with the entire player pool to play
with who they want.

(Also, this was said a while before, but obviously there will be a league
mode where all players have access to all heroes)

**Comment by CarpeSwag** [comment]
Yeah that's what I was saying. Having it run the servers and collect player data for like 1-2 months will rule out a lot of potential issues, then for a week or two we have flashing notifications on the mod ingame or something and on the forum about the upcoming change.

I'm not opposed to gating entirely, I just think trying to sell it off the bat to DOTA 2 players is going to be a pain in the ass that's not worth it.

And nah fam, itsk. I have a bad habit of editing in half my post because I prematurely press comment.

**Comment by Arhowk** [comment]
It's okay, I know some people have trouble with doing things prematurely.

I suppose what has to be done has to be done. It is a gentle balance
between pleasing players and introducing mechanics that increase replay
ability

**Comment by Arhowk** [comment]
Did you ever finish the hero glows?

**Comment by CarpeSwag** [comment]
Tbh I totally forgot about those. I'll add them sometime this week hopefully.
